modules/nw-rfhe-subscribing-redfish-events.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * networking/using-rfhe.adoc
+:_content-type: PROCEDURE
+[id="nw-rfhe-troubleshooting_{context}"]
+= Subscribing to Redfish Events
```

Comment:
There is a doc for redfish event subsciption in progress. The section can be replaced by a reference to this doc once it is ready:
https://github.com/openshift/openshift-docs/pull/42700